components/x-follow-button/src/Input.jsx

```diff
@@ -0,0 +1,16 @@
+import { h } from '@financial-times/x-engine';
```

Comment:
if it's just passing all the props through, i don't think we need this wrapper component for `<input>`, unless you want to add support for consumers to override it?